tensorboard/webapp/metrics/views/metrics_container_test.ts

```diff
@@ -12,87 +12,25 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-import {DebugElement, NO_ERRORS_SCHEMA} from '@angular/core';
+import {NO_ERRORS_SCHEMA} from '@angular/core';
 import {TestBed} from '@angular/core/testing';
 import {By} from '@angular/platform-browser';
-import {NoopAnimationsModule} from '@angular/platform-browser/animations';
-import {Action, Store} from '@ngrx/store';
-import {MockStore, provideMockStore} from '@ngrx/store/testing';
-import {State} from '../../app_state';
-import {
-  getIsTimeSeriesPromotionEnabled,
-  getPromoteTimeSeries,
-} from '../../selectors';
-import {metricsPromoDismissed, metricsPromoGoToScalars} from '../actions';
 import {MetricsDashboardContainer} from './metrics_container';
-import {MetricsPromoNoticeComponent} from './metrics_promo_notice_component';
-import {MetricsPromoNoticeContainer} from './metrics_promo_notice_container';
 
 describe('metrics view', () => {
-  let store: MockStore<State>;
-
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      imports: [NoopAnimationsModule],
-      declarations: [
-        MetricsDashboardContainer,
-        MetricsPromoNoticeContainer,
-        MetricsPromoNoticeComponent,
-      ],
-      providers: [provideMockStore()],
+      declarations: [MetricsDashboardContainer],
+      providers: [],
       // Ignore errors from components that are out-of-scope for this test:
       // 'runs-selector'.
       schemas: [NO_ERRORS_SCHEMA],
     }).compileComponents();
-
-    store = TestBed.inject<Store<State>>(Store) as MockStore<State>;
-    store.overrideSelector(getIsTimeSeriesPromotionEnabled, false);
-    store.overrideSelector(getPromoteTimeSeries, false);
   });
 
   it('renders', () => {
     const fixture = TestBed.createComponent(MetricsDashboardContainer);
     fixture.detectChanges();
-
     expect(fixture.debugElement.query(By.css('runs-selector'))).not.toBeNull();
```

Comment:
This may not be in the scope of this PR but, should we also test that the metrics-main-view renders as well.

Reply:
To be honest I was considering deleting this test since there isn't really any logic remaining. I kept this test just as a sanity that it renders anything at all. I don't think it really matters what it looks up - just that it can look up anything.